CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+## v1.20.0 (06/08/2024)
+
+**Common**
+* Updated links and filenames in the documentation.
+* Improved the CI scripts and pipelines.
+
+**olp-cpp-sdk-authentication**
+* Fixed the DateTime parser to respect DST timezones on iOS.
+* Fixed determining default credentials file absolute path on Windows for new distributions.
+
+**olp-cpp-sdk-core**
+* Extend logging with `olp::logging::LogContext` and `olp::logging::ScopedLogContext` to give users ability to manage logs better.
+* Added logging contexts usage to `olp::http::NetworkCurl` class.
+* Added a new error code `olp::client::ErrorCode::NoSpaceLeft` to signal shortage of available space on the target device.
+* To be able to propagate actual operation error, extended `olp::cache::KeyValueCache` and `olp::cache::DefaultCache` with methods `Read`, `Write`, `Delete` and `DeleteByPrefix`, which do the same actions as `Put`, `Get`, `Remove` and `RemoveKeysWithPrefix` correspondently, but return `olp::cache::OperationOutcome`.
+* Extended `olp::client::OlpClientSettings` with a new setting `propagate_all_cache_errors` to allow users enabling the propagation of all cache errors.
+* Added a new way to serialize a byte vector to JSON. 
```

Comment:
Shouldn't we say what new way that is? Or maybe it isn't really possible to describe it in a couple of words?

Reply:
Rephrased it.